src/libNode/MicroBlockProcessing.cpp

```diff
@@ -93,6 +93,15 @@ bool Node::ProcessMicroblockConsensus(const vector<unsigned char>& message,
     if (cv_processConsensusMessage.wait_for(
             cv_lk, std::chrono::seconds(CONSENSUS_MSG_ORDER_BLOCK_WINDOW),
             [this, message, offset]() -> bool {
+                lock_guard<mutex> g(m_mutexConsensus);
+                if (m_mediator.m_lookup->m_syncType != SyncType::NO_SYNC)
+                {
+                    LOG_GENERAL(WARNING,
+                                "The node started the process of rejoining, "
+                                "Ignore rest of "
+                                "consensus msg.")
+                    return false;
+                }
```

Comment:
I think we should add this to every cases of consensus. In case rejoin as DS was called in the future

Reply:
Implemented